default_positions:
  "0":    0.00
  "1":    0.00
  "2":    0.25
  "3":    0.60
  "4":    0.35
  "5":   -0.00
  "6":    0.00
  "7":   -0.25
  "8":   -0.60
  "9":   -0.35